src/test/java/com/wikia/webdriver/testcases/desktop/createawikitests/CreateWikiTestsLoggedOutUser.java

```diff
@@ -39,6 +39,9 @@ public void loggedOutUserCanCreateNewWiki() {
     new WikiFactory().setIsTestWiki(article.getWikiID(), true);
 
     article.verifyWikiTitleHeader(wikiName);
-    article.verifyUserLoggedIn(credentials.userName10);
+    // Caused by: https://wikia-inc.atlassian.net/browse/PLATFORM-3829
+    // User wont be logged in after the migration is finished, since we are not going to sync access token over in CNW
+    // TODO: Enable once wikia.com -> fandom.com migration is complete
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1135)

Reply:
@ludwikkazmierczak  to repo przetrwalo sporo dluzej niz moglem sadzic, ale widze ze juz umarlo :D